app/ui/client/views/404/roomNotFound.js

```diff
@@ -20,4 +20,13 @@ Template.roomNotFound.helpers({
 	customErrorMessage() {
 		return this.error.reason;
 	},
+	headerMessage() {
+		const { type } = Template.currentData();
+
+		if (type === 'd') {
+			return 'User_not_found';
```

Comment:
Instead of using a `if` here, you can use a ternary operator like:  `return type === 'd' ? 'User_not_found' : 'Room_not_found';
`